pkg/fixtures/writer_s3.go

```diff
@@ -0,0 +1,108 @@
+package fixtures
+
+import (
+	"context"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/aws/aws-sdk-go/aws"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type BlobFixturesSettings struct {
+	BasePath   string
+	ConfigName string
+}
+
+type blobFixtureWriter struct {
+	logger      mon.Logger
+	batchRunner *blob.BatchRunner
```

Comment:
should we accept an interface here instead?

Reply:
That'd be `Module` interface or `RunnableCallable` but why? I say it's better this way since it makes it clearer.

Reply:
But it makes it untestable unit test wise. Feel free to create an interface for the BatchRunner, because we would not want any kernel.Module or stream.RunnableCallback in here (though their interface may -by pure randomness- be the same).

Reply:
Applied.